setup.py

```diff
@@ -58,6 +58,11 @@ def read_file(filename, encoding='utf8'):
 else:
     install_requires.append('mock')
 
+if sys.version_info < (2, 7, 9):
```

Comment:
this should not have been included in this PR; `ndg-httpsclient` and `pyasn1` were not previously explicit dependencies, and I don't think they should be. @Natim, could you please revert?